src/components/MissingDataFeedback/Message.tsx

```diff
@@ -0,0 +1,24 @@
+import { Alert, Box } from '@chakra-ui/react'
+import React from 'react'
+import { FaInfoCircle } from 'react-icons/fa'
+import { IconCircle } from 'components/IconCircle'
+import { Text } from 'components/Text'
+
+export const MissingDataMessage: React.FC<{
+  tkey: 'assetUnavailable' | 'priceHistoryUnavailable' | 'balanceHistoryUnavailable' | 'loading'
```

Comment:
Nitpick but wdyt about extracting this to a type? It would be easier to read to me

Reply:
Also, I really think that passing this key only is adding too much complexity, I would pass the complete key everytime I use this component and type this as a string

Also, `tkey` looks wrong to me, what is `t`? I would call it `translation`

Reply:
please implement both of @NeOMakinG's suggestions for readability